packages/business-rules/src/constants.js

```diff
@@ -13,6 +13,11 @@ const APPEAL_ID = {
 	FAST_TRACK_TREES: '1011'
 };
 
+const TYPE_CODE = {
+	HOUSEHOLDER: 'HAS',
```

Comment:
think these exist in common/src/constants already

Reply:
Have changed to use the ones in common.

Will push up in just a min